### ISSUE TITLE
Status Images for Cron Jobs

### ISSUE THREAD

**Issue by mnonnenmacher** [travis-ci/travis-ci#8945]
Currently the status images always shows the latest build for a branch. This is not ideal when the full test set is only run in a cron job for performance reasons. For such a scenario it would be nice to have an additional parameter in the URL of the status image to show the status of the last cron triggered build.

**Comment by sschuberth** [comment]
I believe this has not been solved yet and thus should be kept open.

**Comment by nickanderson** [comment]
Right, this hasn't been solved. Build badges seem to track the latest build on the branch with no way to target cron so that you are displaying a badge for something like the nightly build only

**Comment by craigcomstock** [comment]
A wild guess. I'll try and install and test some time.

https://github.com/travis-ci/travis-api/pull/1211
